### PR TITLE
Updated hvac.py with Bacpypes working

### PR DESCRIPTION
**Pull request by sguduguntla**

### CODE REVIEW

File: proto/iot.proto [hunk 1]
```diff
@@ -53,6 +53,7 @@ message XBOSIoTDeviceState {
     VAV vav = 12;
     Damper damper = 13;
     Fan fan = 14;
+
```
Comment (gtfierro) [inline proto/iot.proto]: Can you remove this whitespace to keep the file style consistent?

File: python/pyxbos/pyxbos/drivers/hvac/hvac.py [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+                                else:
+                                    props[prop] = 0
+
+                    data[equipclass.lower()] = getattr(hvac_pb2, equipclass)(**props)
+                    data['time'] = int(time.time()*1e9)
+
+                    msg = xbos_pb2.XBOS(XBOSIoTDeviceState = iot_pb2.XBOSIoTDeviceState(**data))
+                    self.report(eq, msg) # Put name of equipment here as the first argument
+
+def get_point_value_mapping(device_points):
```
Comment (gtfierro) [inline python/pyxbos/pyxbos/drivers/hvac/hvac.py]: Can you write some documentation explaining what this method is doing? It looks like it is just reading and parsing strings; is that how we get values out of BAC0?

File: python/pyxbos/pyxbos/drivers/hvac/hvac.py [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+            point_value_map = get_point_value_mapping(dev.points)
+
+            #AHU-1.DPR-O Damper point MISSING from Bacnet points
+
+            output = {}
+            for equipclass in self.point_class_map:
+                if equipclass not in ['Boiler', 'Chilled_Water_Pump', "Hot_Water_Pump"]:
+                    if equipclass not in output:
+                        output[equipclass] = {}
+
```
Comment (gtfierro) [inline python/pyxbos/pyxbos/drivers/hvac/hvac.py]: Can you write some documentation explaining how the parsing logic works? It seems like some of this is hardcoded to the particular test building, but ideally this is logic that can generalize to more buildings. I realize this is difficult without having another set of points that we can use, but its worth starting to make that effort now in anticipation of future changes

File: python/pyxbos/pyxbos/drivers/hvac/point_to_class.json [hunk 4]
```diff
@@ -0,0 +1,1318 @@
+{
```
Comment (gtfierro) [inline python/pyxbos/pyxbos/drivers/hvac/point_to_class.json]: Can you move these JSON files to a subdirectory for the building and determine the subdirectory to use from the driver configuration? That way we can more easily switch between buildings

File: python/pyxbos/pyxbos/drivers/hvac/point_to_value.json [hunk 5]
```diff
@@ -0,0 +1,479 @@
+{
```
Comment (gtfierro) [inline python/pyxbos/pyxbos/drivers/hvac/point_to_value.json]: Is this file used in the driver? Or just for development? If its not used in the driver, we can remove it

File: python/pyxbos/pyxbos/drivers/hvac/point_values_by_equipment.json [hunk 6]
```diff
@@ -0,0 +1,429 @@
+{
```
Comment (gtfierro) [inline python/pyxbos/pyxbos/drivers/hvac/point_values_by_equipment.json]: Is this file used in the driver? If not, we should remove it

File: python/pyxbos/pyxbos/drivers/hvac/hvac.py (hunk 3 above)
Comment (sguduguntla) [reply python/pyxbos/pyxbos/drivers/hvac/hvac.py]: I just made an excluded_equipment list in get_point_class_dict()

File: python/pyxbos/pyxbos/drivers/hvac/point_values_by_equipment.json (hunk 6 above)
Comment (sguduguntla) [reply python/pyxbos/pyxbos/drivers/hvac/point_values_by_equipment.json]: No, but I moved all the json files into a subdirectory for orinda-public-library

File: python/pyxbos/pyxbos/drivers/hvac/point_to_value.json (hunk 5 above)
Comment (sguduguntla) [reply python/pyxbos/pyxbos/drivers/hvac/point_to_value.json]: No, but I moved all the json files into a subdirectory for orinda-public-library for future reference.

**Pull request merged**